modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/impl/MetaStorageCompactionTrigger.java

```diff
@@ -319,4 +322,33 @@ private void cancelLastScheduledFutureBusy() {
             lock.unlock();
         }
     }
+
+    private void startCompactionOnRecoveryAsync() {
```

Comment:
Why is it `Async`? It does not return a future, it just does a compaction in the background